test/rfqt_test.ts

```diff
@@ -0,0 +1,531 @@
+                    [
+                        {
+                            endpoint: 'https://mock-rfqt1.club',
+                            responseData: ganacheZrxWethOrder1,
+                            responseCode: 200,
+                            requestApiKey: 'koolApiKey1',
+                            requestParams: mockedApiParams,
+                        },
+                    ],
+                    async () => {
+                        const appResponse = await request(app)
+                            .get(
+                                `${SWAP_PATH}/quote?buyToken=ZRX&sellToken=WETH&sellAmount=${sellAmount.toString()}&takerAddress=${takerAddress}&intentOnFilling=true&excludedSources=Uniswap,Eth2Dai,Kyber,LiquidityProvider`,
+                            )
+                            .set('0x-api-key', 'koolApiKey1')
+                            .expect(HttpStatus.OK)
+                            .expect('Content-Type', /json/);
+
+                        const responseJson = JSON.parse(appResponse.text);
+                        expect(responseJson.orders.length).to.equal(1);
+                        expect(responseJson.orders[0]).to.eql(ganacheZrxWethOrder1);
+                    },
+                );
+            });
+            it('should not include an RFQ-T order when intentOnFilling === false', async () => {
+                const sellAmount = new BigNumber(100000000000000000);
+
+                const wethContract = new WETH9Contract(contractAddresses.etherToken, provider);
+                await wethContract.deposit().sendTransactionAsync({ value: sellAmount, from: takerAddress });
+                await wethContract
+                    .approve(contractAddresses.erc20Proxy, sellAmount)
+                    .sendTransactionAsync({ from: takerAddress });
+
+                const mockedApiParams = {
+                    sellToken: contractAddresses.etherToken,
+                    buyToken: contractAddresses.zrxToken,
+                    sellAmount: sellAmount.toString(),
+                    buyAmount: undefined,
+                    takerAddress,
+                };
+                return rfqtMocker.withMockedRfqtIndicativeQuotes(
+                    [
+                        {
+                            endpoint: 'https://mock-rfqt1.club',
+                            responseData: rfqtIndicativeQuoteResponse,
+                            responseCode: 200,
+                            requestApiKey: 'koolApiKey1',
+                            requestParams: mockedApiParams,
+                        },
+                    ],
+                    async () => {
+                        const appResponse = await request(app)
+                            .get(
+                                `${SWAP_PATH}/quote?buyToken=ZRX&sellToken=WETH&sellAmount=${sellAmount.toString()}&takerAddress=${takerAddress}&intentOnFilling=false&excludedSources=Uniswap,Eth2Dai,Kyber,LiquidityProvider&skipValidation=true`,
```

Comment:
nit: `excludedSources=${EXCLUDED_SOURCES}` so we don't have to adjust this many times.